packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.ts

```diff
@@ -146,50 +218,106 @@ const useRootStyles = makeStyles({
   },
 });
 
+const useInputClassName = makeResetStyles({
+  gridColumnStart: '1',
+  gridColumnEnd: '2',
+  gridRowStart: '1',
+  gridRowEnd: '3',
+  outlineStyle: 'none',
+  border: '0',
+  padding: '0',
+  color: tokens.colorNeutralForeground1,
+  backgroundColor: 'transparent',
+  ...typographyStyles.body1,
```

Comment:
If you change this to inherit, then you don't need to have a `small` class that sets the typography styles as well:
```suggestion
  // Inherit typography styles from root
  fontFamily: 'inherit',
  fontSize: 'inherit',
  fontWeight: 'inherit',
  lineHeight: 'inherit',
```